### PR TITLE
Add Sun+moon glyph

### PR DESCRIPTION
**Pull request by jashieh**
Adds the following glyphs:

image
Screen Shot 2021-02-17 at 2 19 21 PM

### CODE REVIEW

File: src/icons/glyphs/svgs/SunMoonGlyph.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <svg
+      width={48}
+      height={48}
+      viewBox="0 0 48 48"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <mask
```
Comment (jashieh) [inline src/icons/glyphs/svgs/SunMoonGlyph.tsx:13]: What content should go inside the SVG tags here? I just copied the path from the `sun-moon-glyph.svg` file.

File: src/svgs/glyphs/sun-moon-glyph-secondary.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  </mask>
+  <g mask="url(#mask0)">
+    <path d="M-5.7016 -5.73889H40.5078V40.4705H-5.7016V-5.73889Z" fill="currentColor"/>
+  </g>
+</svg>
```
Comment (michaeljaltamirano) [inline src/svgs/glyphs/sun-moon-glyph-secondary.svg:10]: I've added some [additional instructions in Notion](https://www.notion.so/curology/Working-with-SVGs-c70a743abbe144f1898e9c1095b5b267#ae5c0648f0a2440d8ae87eb4ff2f337c) about this, but can remove anything related to masks:

```suggestion
<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
<title>Sun Moon Glyph</title>
<path d="M35.927 37.2565C38.7721 35.1339 39.3259 31.0635 37.3748 28.5075C36.8642 27.8387 36.2755 27.2993 35.6045 26.8871C36.7826 28.8727 36.9075 30.9822 35.9457 33.1577C35.7782 33.5365 35.3353 33.7078 34.9564 33.5403C34.5776 33.3728 34.4063 32.9299 34.5738 32.5511C35.6244 30.175 35.0667 27.977 32.8144 25.8069C32.2822 25.2941 32.7342 24.4019 33.4625 24.5277C35.5751 24.8928 37.2882 25.9221 38.5671 27.5974C41.0233 30.815 40.3402 35.8355 36.8239 38.4587C33.6585 40.8202 28.6993 40.2925 26.0022 36.4077C25.5596 35.7698 26.2589 34.9621 26.9535 35.3089C29.4223 36.5415 31.8881 35.905 33.493 34.0718C33.7658 33.7601 34.2397 33.7287 34.5513 34.0015C34.863 34.2743 34.8944 34.7482 34.6216 35.0598C33.2281 36.6516 31.2895 37.5393 29.2048 37.4392C31.3616 38.7743 34.0538 38.6539 35.927 37.2565Z" fill="#2D2A26"/>
<path d="M18.4999 22.7499C20.8471 22.7499 22.7499 20.8471 22.7499 18.4999C22.7499 16.1527 20.8471 14.2499 18.4999 14.2499C16.1527 14.2499 14.2499 16.1527 14.2499 18.4999C14.2499 20.8471 16.1527 22.7499 18.4999 22.7499ZM18.4999 21.2499C16.9812 21.2499 15.7499 20.0187 15.7499 18.4999C15.7499 16.9811 16.9812 15.7499 18.4999 15.7499C20.0187 15.7499 21.2499 16.9811 21.2499 18.4999C21.2499 20.0187 20.0187 21.2499 18.4999 21.2499ZM8.49994 19.2499C8.08573 19.2499 7.74994 18.9141 7.74994 18.4999C7.74994 18.0857 8.08573 17.7499 8.49994 17.7499H12.4999C12.9142 17.7499 13.2499 18.0857 13.2499 18.4999C13.2499 18.9141 12.9142 19.2499 12.4999 19.2499H8.49994ZM24.4999 19.2499C24.0857 19.2499 23.7499 18.9141 23.7499 18.4999C23.7499 18.0857 24.0857 17.7499 24.4999 17.7499H28.4999C28.9142 17.7499 29.2499 18.0857 29.2499 18.4999C29.2499 18.9141 28.9142 19.2499 28.4999 19.2499H24.4999ZM11.9592 26.1013C11.6663 26.3942 11.1914 26.3942 10.8985 26.1013C10.6056 25.8084 10.6056 25.3335 10.8985 25.0406L13.727 22.2122C14.0199 21.9193 14.4947 21.9193 14.7876 22.2122C15.0805 22.5051 15.0805 22.98 14.7876 23.2728L11.9592 26.1013ZM23.2729 14.7876C22.98 15.0805 22.5051 15.0805 22.2122 14.7876C21.9194 14.4947 21.9194 14.0198 22.2122 13.7269L25.0407 10.8985C25.3336 10.6056 25.8084 10.6056 26.1013 10.8985C26.3942 11.1914 26.3942 11.6662 26.1013 11.9591L23.2729 14.7876ZM26.1013 25.0406C26.3942 25.3335 26.3942 25.8084 26.1013 26.1013C25.8084 26.3942 25.3336 26.3942 25.0407 26.1013L22.2122 23.2728C21.9194 22.98 21.9194 22.5051 22.2122 22.2122C22.5051 21.9193 22.98 21.9193 23.2729 22.2122L26.1013 25.0406ZM14.7876 13.7269C15.0805 14.0198 15.0805 14.4947 14.7876 14.7876C14.4947 15.0805 14.0199 15.0805 13.727 14.7876L10.8985 11.9591C10.6056 11.6662 10.6056 11.1914 10.8985 10.8985C11.1914 10.6056 11.6663 10.6056 11.9592 10.8985L14.7876 13.7269ZM19.2499 28.4999C19.2499 28.9141 18.9142 29.2499 18.4999 29.2499C18.0857 29.2499 17.7499 28.9141 17.7499 28.4999V24.4999C17.7499 24.0857 18.0857 23.7499 18.4999 23.7499C18.9142 23.7499 19.2499 24.0857 19.2499 24.4999V28.4999ZM19.2499 12.4999C19.2499 12.9141 18.9142 13.2499 18.4999 13.2499C18.0857 13.2499 17.7499 12.9141 17.7499 12.4999V8.49988C17.7499 8.08566 18.0857 7.74988 18.4999 7.74988C18.9142 7.74988 19.2499 8.08566 19.2499 8.49988V12.4999Z" fill="#2D2A26"/>
</svg>
```

File: src/svgs/glyphs/sun-moon-glyph.svg [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (michaeljaltamirano) [inline src/svgs/glyphs/sun-moon-glyph.svg:1]: Ditto the above suggestion here

File: src/icons/glyphs/svgs/SunMoonGlyph.tsx (hunk 1 above)
Comment (michaeljaltamirano) [reply src/icons/glyphs/svgs/SunMoonGlyph.tsx:13]: This file is auto-generated from `yarn run build:icons:index` so we shouldn't be adding anything here, let me know if anything about the steps are confusing.

**Review by michaeljaltamirano — APPROVED** [review]

**Review by daigof — APPROVED** [review]
It looks perfect you got it right. The only thing to comment is to Always make sure to run storybook and check the new icon/glyph. Not only its design but its dimensions: in the case of glyphs it should always be perfect 48x48

**Pull request merged**